network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -84,16 +84,17 @@ def generate_image_field():
     image_id = choice(Image.objects.all()).id
     alt_text = " ".join(fake.words(nb=5))
     caption = " ".join(fake.words(nb=5))
-    caption_url = fake.url(schemes=["https"])
+    caption_external_url = [
```

Comment:
Since this StreamField provider file generates JSON data for StreamBlocks, we are simulating JSON data that represents a ListBlock housing a LinkWithoutLabelBlock linking to a random external URL.

As we move along the epic [TP1-60](https://mozilla-hub.atlassian.net/browse/TP1-60) it might be worth moving the generation of a LinkWithoutLabelBlock data to a function of its own!